core/src/test/java/com/bumble/appyx/core/children/ChildAwareTestBase.kt

```diff
@@ -27,6 +28,7 @@ import org.junit.Rule
 import kotlin.coroutines.EmptyCoroutineContext
 import kotlin.reflect.KClass
 
+@ExperimentalCoroutinesApi
```

Comment:
we can remove it add add `@OptIn(ExperimentalCoroutinesApi::class)` to `MainDispatcherRule`